tests/PrestaShopBundle/Service/Hook/HookDispatcherTest.php

```diff
@@ -80,24 +80,27 @@ public function testAddListenerRendering()
     {
         $kernel = $this->createKernel();
         $kernel->boot();
-        $hookDisptacher = $kernel->getContainer()->get('prestashop.hook.dispatcher');
+        $hookDispatcher = $kernel->getContainer()->get('prestashop.hook.dispatcher');
+
+        $hookDispatcher->addListener('test_test_2', array($this, 'listenerCallback2'));
+        $hookDispatcher->addListener('test_test_2', array($this, 'listenerCallback2b'));
+        $event = $hookDispatcher->dispatch('test_test_2', new RenderingHookEvent());
 
-        $hookDisptacher->addListener('test_test_2', array($this, 'listenerCallback2'));
-        $hookDisptacher->addListener('test_test_2', array($this, 'listenerCallback2b'));
-        $event = $hookDisptacher->dispatch('test_test_2', new RenderingHookEvent());
         $this->assertArraySubset(array(
-            'listenerCallback2' => "result_test_2",
-            'overriden_listener_name' => "result_test_2b"
+            'listenerCallback2' => ["result_test_2"],
```

Comment:
double quote :nauseated_face: 
Should we use cs fixer on test directory?

Reply:
it is done already imo, it's not part of our coding styles